src/client.js

```diff
@@ -182,6 +182,10 @@ function keyFromRecoverySession(session, decryptionKey) {
  * Optional. Whether to allow a fallback ICE server should be used for negotiating a
  * WebRTC connection if the homeserver doesn't provide any servers. Defaults to false.
  *
+ * @param {boolean} [opts.usingE2EProxy]
```

Comment:
Perhaps this is a bit more general?

```suggestion
 * @param {boolean} [opts.usingExternalCrypto]
```

Reply:
Yeah, works for me.